docs/testcases/UI/Billing/762/762_7.md

```diff
@@ -0,0 +1,47 @@
+| :---: | --- | --- |
+| 1 | Login as admin user from the prerequisites | |
+| 2 | Open the **Billing** page | |
+| 3 | Click **Storages** tab in the **Quotas** section |  |
+| 4 | Click **Add quota** button for ***Billing centers*** section | ***Create storages quota*** modal opens that contains: <li> ***Billing center*** field <li> ***Quota*** field, <li> ***Quota period*** dropdown list with values `per month`, `per quarter`, `per year` (default value is `per month`) <li> ***Actions*** section that contains ***Threshold*** and ***Action*** fields and **Add Action** button <li> **Cancel** and **Save** buttons. |
+| 5 | Click into the ***Billing center*** field and select `<Billing_center>` from the prerequisites | | 
+| 6 | Specify follow values for quota <li> *Quota* - `<billing1>` <li> quota period -`per quarter` <li> *Threshold* - `70` <li> *Action* - `Read-only mode` | |
+| 7 | Click **Save** button | The following info appears in the ***Overall*** section: <li> `<Billing_center>: <quota>$ per quarter`, where `<quota>` is value specified at step 5 <li> `70%: read-only mode` - pair of threshold-action configured at step 7-9 |
+| 8 | Wait time equals to `<billing_quotas_period>` from the [762_1](762_1.md) case Preparations and refresh page | Yellow circle is shown near the quota created at step 7|
+| 9 | Open the **Library** page | |
+| 10 | Open the `storage1` created at step 2 of Preparations | Storage page opens and is available for editing |
+| 11 | Create `file1.txt` | | 
+| 12 | Open the `fs_storage` created at step 2 of Preparations | Storage page opens and is available for editing |
+| 13 | Logout | |
+| 14 | Login as non-admin user from the prerequisites | |
+| 15 | Repeat steps 10-11, 13 | For both cases Storage page opens and doesn't contain <li> **Create** and **Upload** buttons for whole table <li> **Edit** and **Remove** buttons for each row |
+| 16 | At the **Runs** page, click the run launched at step 6 of the Preparations | |
+| 17 | Wait until the **SSH** hyperlink appears | |
+| 18 | Click the **SSH** hyperlink | |
+| 19 | Execute command: <br> `echo test >> /cloud-data/<storage2_name with path>/test_file1.txt`, <br>where `<storage2_name with path>` is the full path to the `storage2` created at step 2 of the Preparations |  <li>`test_file1.txt` file isn't created <li> Response `Read-only file system` is shown |
+| 20 | Execute command: <br> `pipe storage cp cp://<storage1>/file1.txt cp://<storage2>/`, <br> where `<storage1>` and `<storage2>` are the storages created at step 2 of the Preparations |  <li>`file1.txt` file isn't copied <li> Response `Read-only file system` is shown |
+| 21 | Execute command: <br> `echo test >> /cloud-data/<fs_storage>/test_file1.txt`, <br> where `<fs_storage>` is the full path to the `fs_storage` created at step 2 of the Preparations |  <li>`test_file1.txt` file isn't created <li> Response `Read-only file system` is shown |
+
+**After:**
+- Remove quota created at step 7 
```

Comment:
Add row:
- Stop the run launched at step 6 of the case preparations

Reply:
Done